modules/javafx.graphics/src/main/java/javafx/scene/text/Text.java

```diff
@@ -1986,8 +1986,8 @@ public Object queryAccessibleAttribute(AccessibleAttribute attribute, Object...
             }
             case LINE_FOR_OFFSET: {
                 int offset = (Integer)parameters[0];
-                if (offset > getTextInternal().length()) return null;
                 TextLine[] lines = getTextLayout().getLines();
+                if (offset > getTextInternal().length()) return lines.length;
```

Comment:
A couple questions:

1. This will return an offset that represents the position just past the end of the list of lines. I presume this is OK?
2. The `LINE_START` and `LINE_END` cases also can return null. Should a similar fix be applied for those cases?

Reply:
When rechecked it looks wrong to change the return value/type. At several instances the return value `null` is used to assume that the control is `TextField`.
1. https://github.com/openjdk/jfx/blob/f95f09f5af561e3a6f3804a4f54aff83e9d2bb0b/modules/javafx.graphics/src/main/java/com/sun/glass/ui/win/WinTextRangeProvider.java#L183
2. https://github.com/openjdk/jfx/blob/f95f09f5af561e3a6f3804a4f54aff83e9d2bb0b/modules/javafx.graphics/src/main/java/com/sun/glass/ui/win/WinTextRangeProvider.java#L197

So changed to an alternative way of passing correct offset: If start(offset) is larger than length of text then pass length of text.